### DISCUSSION TITLE
Two Complete Separate Instances

### DISCUSSION THREAD

**Discussion by racorbin2010** [FreeTubeApp/FreeTube#4452] [Q&A]
If this is answered elsewhere, I couldn't find it and I apologize. I would like to have two separate instances of the TERRIFIC FreeTube. Each with different Playlists, History, Settings, etc. I suspect NO because everything is stored in %APPDATA% so the two instances seem to step on each other. Any comments highly appreciated!

**Comment by DejayRezme** [comment]
I'm not sure but maybe windows has a feature to start an app "for another user" but on your desktop which would allow this

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
I was able to install FT in 2 different user accounts with separate Playlists, History, Settings etc.

On install FT asks you where to install it and you can choose between `only this user` and `for all users`. I selected `only this user` on both accounts.

Hope this helps :)

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
@racorbin2010 did this help?

**Comment by DejayRezme** [comment]
Not the same as OPs request, but after using freetube for a while now I've started thinking that it would be nice have two separate subscription feeds. Basically for mental health reasons. I am subscribed to news and politics and "serious" things and sometimes it's hard to look at this "fresh horror of the day" and then look at my frivolous channels for entertainment or edutainment. Like have a subscription list for news and "other".

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
You can use profiles for this

**Reply by DejayRezme** [reply]
Thank you, awesome!
